runtime/parachains/src/initializer.rs

```diff
@@ -63,61 +61,65 @@ impl<BlockNumber: Default + From<u32>> Default for SessionChangeNotification<Blo
 }
 
 #[derive(Encode, Decode)]
-struct BufferedSessionChange {
+pub struct BufferedSessionChange {
```

Comment:
why having make it public ? can't we have this kept private, and the storage kept private too ?
```suggestion
struct BufferedSessionChange {
```